packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -201,95 +199,90 @@ module.exports = (
 
         const { flowId, flowBeginTime } = await request.app.metricsContext;
 
-        return customs
-          .check(request, sessionToken.email, 'recoveryEmailResendCode')
-          .then(setVerifyCode)
-          .then(setVerifyFunction)
-          .then(() => {
-            if (!sendEmail) {
-              return;
-            }
+        await customs.check(
+          request,
+          sessionToken.email,
+          'recoveryEmailResendCode'
+        );
+        await setVerifyCode();
+        setVerifyFunction();
```

Comment:
await?

Reply:
Ah, nevermind, I found setVerifyFunction which isn't async, odd that it was in a then before.